src/openrct2/config/Config.cpp

```diff
@@ -380,6 +380,7 @@ namespace Config
             model->known_keys_only = reader->GetBoolean("known_keys_only", false);
             model->log_chat = reader->GetBoolean("log_chat", false);
             model->log_server_actions = reader->GetBoolean("log_server_actions", false);
+            model->pause_server = reader->GetBoolean("pause_server", false);
```

Comment:
I think this option should be renamed to make its meaning clearer.
I'd suggest `pause_server_if_no_clients`.

Reply:
What's about merging this with @janisozaur suggestion for bots. Maybe `pause_server_at_client`?

Reply:
Hmm, I don't think that's clear enough. Moreover, it suggests the server is paused clientside, which is not what's happening. I think @Gymnasiast's suggestion is better.

Reply:
@tobiaskohlbau 
Currently there are no bots, and I doubt they'll come soon - not after the scripting support, I'd say. I would argue it's best to follow the YAGNI principle and keep the feature as you have it currently, only fixing the option name and code style.